### DISCUSSION TITLE
when npm start

### DISCUSSION THREAD

**Discussion by 4xsay** [community/community#18399] [Projects and Issues] [answered]
Error: tunneling socket could not be established, cause=connect ECONNREFUSED 127.0.0.1:7890

**Comment by sindrimt** [comment]
https://stackoverflow.com/questions/18419144/npm-not-working-read-econnreset 's following solution could solve your problem
npm config get proxy
npm config rm proxy

**Comment by zynowhere** [comment, marked as answer]
enable proxy？

**Reply by miteokaku** [reply]
No
